lib/mirage.ml

```diff
@@ -26,6 +26,8 @@ module StringSet = struct
     List.iter (fun e -> s := add e !s) l;
     !s
 
+  let to_list s = fold (fun x xs -> x :: xs) s []
```

Comment:
Actually it doesn't seem to be used by the PR (also my suggestion would be in fact the reverse of the current `to_list`).